packages/libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -443,7 +443,8 @@ export class SolanaWeb3Manager {
         console.debug('Getting associated token account for', pubkey.toBase58())
         const tokenAccountAddress = splToken.getAssociatedTokenAddressSync(
           this.mints.audio,
-          pubkey
+          pubkey,
+          true
```

Comment:
This prevents the method from throwing when the account is a PDA (which is the case when the user bank has not been created yet)